assets/img/logo.svg

```diff
@@ -0,0 +1 @@
-1" d="M146,302.87c1.81,0,3.52-.09,5.21.05a2.53,2.53,0,0,1,1.56,1.1q9.42,14.19,18.76,28.45a4.12,4.12,0,0,0,1.31,1.38V303h6.61V347.7c-1.85,0-3.61.09-5.35-.05-.48,0-1-.68-1.35-1.16q-9.34-14.14-18.65-28.32a4.09,4.09,0,0,0-1.46-1.43V347.6H146Z"/><path class="cls-1" d="M256.36,303.16c6.83,0,13.51-.39,20.11.11A11.67,11.67,0,0,1,287.34,314c.69,6.49-2,11.07-8,13.52l-.78.34,10.28,19.81c-2.51,0-4.73.07-6.94-.06-.51,0-1.16-.69-1.43-1.22-2.73-5.34-5.43-10.7-8-16.09a2.42,2.42,0,0,0-2.66-1.7c-2.16.14-4.34,0-6.7,0v18.88h-6.67Zm6.7,5.71v14.05a5,5,0,0,0,.64.11c3.3,0,6.6,0,9.9,0,4,0,7-2.68,7.11-6.35.19-4.46-2.24-7.51-6.58-7.76C270.5,308.7,266.84,308.87,263.06,308.87Z"/><path class="cls-1" d="M362.05,329.14h-9.4v-5.49h16.09c-.09,4.6.48,9.19-.79,13.69-1.81,6.45-8.34,10.86-15.76,10.73-8.27-.14-14.92-5.84-15.51-13.74-.38-5-.19-10.13-.21-15.19a19.86,19.86,0,0,1,.29-3.7c1.3-7.13,5.85-11.7,13.2-12.75,9.39-1.34,17.14,4.07,18.62,13.58-2,0-4,.07-5.95-.07-.38,0-.85-.8-1.05-1.32-1.9-5.05-6.62-7.53-11.77-6-3.8,1.1-6.16,3.77-6.32,7.64a144.61,144.61,0,0,0,.17,18.15c.42,4.88,4.55,7.76,9.32,7.47a9.25,9.25,0,0,0,9-8.7C362.13,332.06,362.05,330.71,362.05,329.14Z"/><path class="cls-1" d="M90.1,302.93h6.56v1.93c0,9.25-.08,18.5,0,27.75.11,8.49,7.65,11,13.34,8.69,3.91-1.59,5.39-4.91,5.41-8.92,0-9.14,0-18.27,0-27.4v-2H122a11.12,11.12,0,0,1,.13,1.16c0,9.55.05,19.09,0,28.64A14.82,14.82,0,0,1,111,347.37c-7.35,2-14.91-.53-18.61-6.29A14.32,14.32,0,0,1,90,333.42c-.07-9.9,0-19.8,0-29.7A5.84,5.84,0,0,1,90.1,302.93Z"/><path class="cls-1" d="M198.82,347.71l3.53-9.66c4.08-11.15,8.19-22.3,12.22-33.48a2.17,2.17,0,0,1,2.51-1.74c3.55.11,3.55,0,4.75,3.31q7.17,19.64,14.36,39.27c.26.71.49,1.42.78,2.3-2.14,0-4.08.06-6,0a1.86,1.86,0,0,1-1.3-1c-.84-2.13-1.63-4.29-2.26-6.5-.36-1.28-1-1.59-2.26-1.57-4.78.07-9.56.08-14.33,0a2,2,0,0,0-2.4,1.67c-.61,2.14-1.35,4.26-2.17,6.34a2.05,2.05,0,0,1-1.4,1.09C203,347.79,201.08,347.71,198.82,347.71Zm12.24-14.91h13.87c-2.25-6.56-4.43-12.93-6.62-19.29l-.49.05C215.59,319.9,213.36,326.24,211.06,332.8Z"/><path class="cls-1" d="M423.8,347.71c-2.18,0-4.07.08-5.94,0a2.05,2.05,0,0,1-1.42-1.1c-.87-2.19-1.63-4.42-2.32-6.66a1.65,1.65,0,0,0-1.91-1.32c-4.95,0-9.9.07-14.86,0a1.85,1.85,0,0,0-2.09,1.58c-.8,2.52-1.69,5-2.55,7.51h-7c1.16-3.2,2.26-6.28,3.39-9.36,4.09-11.15,8.23-22.28,12.24-33.46.56-1.55,1.22-2.08,2.85-2,3.27.15,3.27.05,4.38,3.08q7.26,19.78,14.51,39.59C423.28,346.14,423.49,346.8,423.8,347.71Zm-12-14.82c-2.27-6.62-4.45-13-6.64-19.38l-.49,0c-2.25,6.41-4.49,12.82-6.79,19.36Z"/><path class="cls-1" d="M524.81,347.7c-2.27,0-4.31.06-6.35-.05-.39,0-.9-.63-1.07-1.08-.86-2.25-1.67-4.53-2.39-6.83a1.43,1.43,0,0,0-1.65-1.16c-5.13,0-10.26,0-15.39.07a2.16,2.16,0,0,0-1.54,1.17c-.89,2.18-1.64,4.41-2.34,6.66a1.49,1.49,0,0,1-1.75,1.25c-1.82-.08-3.65,0-5.76,0L503,302.92h5.4C513.82,317.75,519.28,332.62,524.81,347.7Zm-18.75-34.29h-.43l-6.81,19.42h13.87Z"/><path class="cls-1" d="M443.18,302.93h6.49v38.73h21.56v5.91H443.18Z"/><path class="cls-1" d="M43.33,303h6.49v38.74H71.39v5.9H43.33Z"/></g></g></svg>
```

Comment:
I put this under `assets/logo` but honestly doesn't really matter...we'll consolidate when we get all the landing pages merged